packages/owl-bot/src/owl-bot.ts

```diff
@@ -658,6 +650,22 @@ const runPostProcessor = async (
     logger.info(`Created ${logLine}`);
   }
 
+  // If the pull request is from a fork and not from allowlist, skip.
+  const forkOwner = opts.head.split('/')[0];
+  if (opts.head !== opts.base && !ALLOWED_FORK_OWNERS.includes(forkOwner)) {
+    logger.info(
+      `head ${opts.head} does not match base ${opts.base}, skipping PR from fork`
+    );
+    await createCheck({
+      text: 'Ignored by Owl Bot because the pull request was created from a fork. See go/owlbot-skip-forks.',
+      summary:
+        'Ignored by Owl Bot because the pull request was created from a fork',
+      conclusion: 'success',
```

Comment:
I don't think we want to mark as success. There should be a `skipped` state I think.

Reply:
@chingor13 I see it's either success or failure.

```
export interface CheckArgs {
  privateKey: string;
  appId: number;
  installation: number;
  pr: number;
  repo: string;
  summary: string;
  conclusion: 'success' | 'failure';
  detailsURL: string;
  text: string;
  title: string;
}
```

and https://github.com/suztomo/repo-automation-bots/blob/913afc1121af735f358e20394b82b94903c83dc4/packages/owl-bot/src/core.ts#L296

Reply:
https://github.com/suztomo/repo-automation-bots/blob/913afc1121af735f358e20394b82b94903c83dc4/packages/owl-bot/src/core.ts#L296 has "skipped".

Now thinking about whether a required check is ok to be skipped (https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/troubleshooting-required-status-checks#handling-skipped-but-required-checks)

Reply:
@chingor13 I think we want "success" in the conclusion. Otherwise we would see "Waiting for status to be reported." We also have precedent in https://github.com/googleapis/repo-automation-bots/blob/cacc73c72cfca7ec1bbadf6319f0b9b58c5edb7f/packages/owl-bot/src/owl-bot.ts#L648-L659 (from Tony's comment in the bug) which returns "success" in conclusion.

Reply:
You can create a conclusion of `skipped` https://docs.github.com/en/rest/checks/runs?apiVersion=2022-11-28#create-a-check-run.  The referenced `interface` is our own helper declaration.

Also, we can confirm that `skipped` conclusion does pass required checks (look at this PR for the skipped GCB presubmits)

Reply:
Updated